### PR TITLE
Edits for the Forest Lake file

### PR DESCRIPTION
**Pull request by JamesWomack-Securian**
Here are some edits to my Forest Lake file.  I only need one :+1:

### CODE REVIEW

File: JamesWomack-Securian-ForestLake.md [hunk 1]
```diff
@@ -10,7 +10,19 @@
...
 
-### Man-made Places
+### Places to visit
 1) St. Peter's Catholic Church
 2) Forest Lake Senior High
+3) Forest Lake Theatre
+4) Hardwood Creek Library
+
+### Forest Lake Golf Courses
+1) Castlewood Golf Course
```
Comment (nate-beaudin) [inline JamesWomack-Securian-ForestLake.md]: How about something for the non-golf fans in FL?

**Review by nate-beaudin — CHANGES_REQUESTED** [review]
Add something for the non-golf fans in FL?

File: JamesWomack-Securian-ForestLake.md (hunk 1 above)
Comment (JamesWomack-Securian) [reply JamesWomack-Securian-ForestLake.md]: @nate-beaudin - They can visit the lake

**Review by nate-beaudin — APPROVED** [review]
OK Fine!!!

**Pull request merged**